regression/esbmc/interval-analysis-function-1/main.c

```diff
@@ -0,0 +1,13 @@
+#include <assert.h>
+
+int foo(int n)
+{
+  // n: [51,60]
```

Comment:
This comment is unclear to me: where do the bounds 51 and 60 come from?

Reply:
ah, it was from another example and I forgot to remove.